components/serializer.rst

```diff
@@ -413,8 +413,81 @@ As for groups, attributes can be selected during both the serialization and dese
 Ignoring Attributes
 -------------------
 
-As an option, there's a way to ignore attributes from the origin object.
-To remove those attributes provide an array via the ``AbstractNormalizer::IGNORED_ATTRIBUTES``
+All attributes are included by default when serializing objects. You have two alternatives to ignore some of those attributes.
+
+* `Option 1: Using @Ignore annotation`_
+* `Option 2: Using the context`_
+
+Option 1: Using ``@Ignore`` annotation
+--------------------------------------
+
+.. configuration-block::
+
+    .. code-block:: php-annotations
+
+        namespace Acme;
+
+        use Symfony\Component\Serializer\Annotation\Ignore;
+
+        class MyObj
+        {
+            public $foo;
+            /**
+             * @Ignore()
+             */
+            public $bar;
+        }
+
+    .. code-block:: yaml
+
+        Acme\MyObj:
+            attributes:
+                foo:
+                    groups:
+                        ignore: false
+                bar:
+                    groups:
+                        ignore: true
```

Comment:
Ignore should probably be listed on annotation level (as a direct child of the attribute name) instead of as being a group.